docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+Watchlists, Feeds, Reports, and IOCs
+====================================
+Watchlists are a powerful feature of Carbon Black Cloud Enterprise EDR. They allow an organization to set-and-forget
+searches on their endpoints' incoming events data, providing the administrator the opportunity to sift through high
+volumes of activity and focus attention on those that matter.
+
+**Note:** Use of these APIs requires that the organization be enabled for Enterprise EDR.  Verify this by logging into
+the Carbon Black Cloud Console, opening the menu in the upper right corner, and checking for an ``ENABLED`` flag
+against the "Enterprise EDR" entry.
+
+All examples here assume that a Carbon Black Cloud SDK connection has been set up, such as with the following code:
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+
+Setting Up a Basic Custom Watchlist
+-----------------------------------
+Creating a custom watchlist that can watch and/or generate alerts requires three steps:
+
+1. Create a report including one or more Indicators of Compromise (IOCs).
+2. Add that report to a watchlist.
+3. Enable alerting on the watchlist.
+
+Creating a Report
++++++++++++++++++
+We create the report, adding one or more IOCs to it:
```

Comment:
I would not use the `we` language as Dev Rel is not doing anything.

Reply:
Untangling this sentence.